src/app/games/lasca/lasca.component.ts

```diff
@@ -62,26 +64,31 @@ export class LascaComponent extends ParallelogramGameComponent<LascaRules,
     private currentMoveClicks: Coord[] = [];
     private capturedCoords: Coord[] = []; // Only the coords capture by active player during this turn
     private legalMoves: LascaMove[] = [];
+    private moveGenerator: LascaMoveGenerator = new LascaMoveGenerator();
 
     public constructor(messageDisplayer: MessageDisplayer) {
         super(messageDisplayer);
         this.hasAsymmetricBoard = true;
         this.rules = LascaRules.get();
         this.node = this.rules.getInitialNode();
-        this.availableMinimaxes = [
-            new LascaControlMinimax('Lasca Control Minimax'),
-            new LascaControlAndDominationMinimax(),
+        this.availableAIs = [
+            new Minimax('Control', this.rules, new LascaControlHeuristic(), new LascaMoveGenerator()),
+            new Minimax('Control and domination',
```

Comment:
Control And Domination
Pour la classe ? (hein David ?)

Reply:
CaD alors !

Reply:
yes, c'est fait !